Simplify PR Template (#1)
oddbird/.github: pull_request_template.md

## Description
_The commit messages say what you did; this should explain why and/or how._
- [ ] Description is in Trello card

## Steps to test/reproduce

> _Please explain how to best reproduce the issue and/or test the changes locally and the pages/URLs/views/states in which to review_

Reviewer: ```suggestion
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._
```
